CHANGELOG.md

```diff
@@ -1,3 +1,56 @@
+## 2.3.0 (Mar 29, 2016)
+### general
+ - Added dynamic config, a new feature to track config file for changes and restart the 
+   pipeline (same process) with updated config changes. This feature can be enabled in two 
+   ways: Passing a CLI long-form option `--auto-reload` or with short-form `-r`. Another 
+   option, `--reload-interval <seconds>` controls how often LS should check the config files 
+   for changes. Alternatively, if you don't start with the CLI option, you can send SIGHUP 
+   or `kill -1` signal to LS to reload the config file, and restart the pipeline ([#4513](https://github.com/elastic/logstash/issues/4513)).
+ - Added support to evaluate environment variables inside the Logstash config. You can also specify a 
+   default if the variable is not defined. The syntax is `${myVar:default}` ([#3944](https://github.com/elastic/logstash/issues/3944)).
+ - Improved throughput performance across the board (up by 2x in some configs) by implementing Event 
+   representation in Java. Event is the main object that encapsulates data as it flows through 
+   Logstash and provides APIs for the plugins to perform processing. This change also enables 
+   faster serialization for future persistence work ([4191](https://github.com/elastic/logstash/issues/4191)).
+ - Add ability to configure custom garbage collection log file using `$LS_LOG_DIR`.
+ - Deprecated `bin/plugin` in favor of `bin/logstash-plugin`. In the next major version `bin/plugin` will 
+   be removed to prevent `PATH` being polluted when other components of the Elastic stack are installed on 
+   the same instance ([#4891](https://github.com/elastic/logstash/pull/4891))
+ - Fixed a bug where new pipeline might break plugins by calling the `register` method twice causing 
+   undesired behavior ([#4851](https://github.com/elastic/logstash/issues/4851))
+ - Make `JAVA_OPTS` and `LS_JAVA_OPTS` work consistently on Windows  ([#4758](https://github.com/elastic/logstash/pull/4758))
+ - Fixed bug where specifying JMX parameters in `LS_JAVA_OPTS` caused Logstash not to restart properly
+   ([#4319](https://github.com/elastic/logstash/issues/4319))
+ - Upgrading plugins with manticore throws an error and sometimes corrupts installation ([#4818](https://github.com/elastic/logstash/issues/4818))
+ - Fixed milestone warning is displayed when pluginpath is used ([#4562](https://github.com/elastic/logstash/issues/4562))
+ - Upgrade to JRuby 1.7.24
+   
+### input
+ - Beats
+   - Verify client certificates against CA ([#8](https://github.com/logstash-plugins/logstash-input-beats/issues/8))
+ - RabbitMQ
+   - Breaking Change: Metadata is disabled by default which was regressing performance. 
```

Comment:
There's a structural problem with this sentence that makes it read like disabling metadata was regressing performance. 

I think you mean to say: Metadata is now disabled by default because it was regressing performance.